sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -815,11 +1087,16 @@ pi_result piContextCreate(const pi_context_properties *properties,
                                              size_t cb, void *user_data1),
                           void *user_data, pi_context *retcontext) {
   pi_result ret = PI_ERROR_INVALID_OPERATION;
+  std::vector<cl_device_id> cl_devices = getClDevices(num_devices, devices);
```

Comment:
Is it ok to have duplicates here? Although I think the helper itself has to be changed, if it's not ok.